reference/orders.v2.oas2.yml

```diff
@@ -2046,6 +2048,7 @@ components:
                 brand: BigCommerce
                 applied_discounts: []
                 product_options: []
+                discounted_total_inc_tax: "0.0000"
```

Comment:
may we have correct value here as well instead of 0.0000 ?

Reply:
Thanks @donald-nguyen-bc It has been updated.